test/corpus/create.txt

```diff
@@ -1,142 +1,137 @@
-================================================================================
+==================
 Create table
-================================================================================
+==================
 
 CREATE TABLE my_schema.my_table (id BIGINT NOT NULL PRIMARY KEY);
 
---------------------------------------------------------------------------------
+---
```

Comment:
Looks like this is undoing the formatting that was done in https://github.com/DerekStride/tree-sitter-sql/pull/77.

You can use `make format` or `tree-sitter test --update` to auto-format these tests. I think it requires a failing test otherwise it skips the formatting, I usually change a test slightly to get it to apply to the file.

Reply:
Thanks for the hint. It is a bit dangerous since in autofills the expected value for the test. But still, good to know